tests/experimental/starry/light_curve_test.py

```diff
@@ -71,26 +71,32 @@ def test_compare_starry(deg, u):
             "central": keplerian.Central(
                 mass=1.3,
                 radius=1.1,
-                map=Map(y=1.0),
+                map=Map(
+                    y=Ylm.from_dense(np.hstack([1.0, np.random.rand(10) * 0.1])),
```

Comment:
I don't think we should use `random` here. If we must, let's use the following instead:

```suggestion
                    y=Ylm.from_dense(np.hstack([1.0, np.random.default_rng(0).uniform(0.0, 0.1, 10)])),
```